forge-gui/res/cardsfolder/upcoming/incessant_provocation.txt

```diff
@@ -0,0 +1,9 @@
+Name:Incessant Provocation
+ManaCost:3 R
+Types:Sorcery
+A:SP$ GainControl | ValidTgts$ Creature | TgtPrompt$ Select target creature | LoseControl$ EOT | AddKWs$ Haste | Untap$ True | RememberTargets$ True | SubAbility$ DBEffect | SpellDescription$ Gain control of target creature until end of turn. Untap that creature. It gains haste until end of turn. It perpetually gains "This creature attacks each combat if able."
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualAbility | Name$ Incessant Provocation's Perpetual Effect | Duration$ Permanent
```

Comment:
Use `RememberObjects$ Targeted` here and you won't need cleanup or `RememberTargets` in line 4